docs/cli/cmdline/script/info.md

```diff
@@ -0,0 +1,49 @@
+---
+title: terramate script info - Command
+description: The script info command gives you information about a script
+
+prev:
+  text: 'Run'
+  link: '/cli/cmdline/run'
+
+next:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+---
+
+# Script Info
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script info` command shows details of all scripts matching the command line argument. The details include:
```

Comment:
An alternative way to describe this would be to say that `script info` lists details about all script definitions with the given name.

The `Stacks` below I would describe as the stacks within the scope of the script, i.e. those stacks it can run on.

Even if you don't introduce the multiple definitions concept, I think it would be more precise to say "The stacks the script can run on"

Reply:
Fixed. much better, thanks